### PR TITLE
Addin Reno Djurs module and documentation

### PR DESCRIPTION
**Pull request by MTrab**
Adding possibility to get schedules from Reno Djurs in Denmark

### CODE REVIEW

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/renodjurs_dk.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+        r = session.get(self._api_url)
+        r.raise_for_status()
+        soup = BeautifulSoup(r.text, "html.parser")
+        container = soup.find("table", {"class": "table"})
+        rows = container.find_all("tr")
+        
+        for row in rows:
+            cells = row.find_all("td")
+            if len(cells) > 4:
```
Comment (5ila5) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/renodjurs_dk.py:47]: I'd prefer `if len(cells) >= 4: continue` to reduce indentation depth

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/renodjurs_dk.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                    icon = ICON_MAP.get("PLAST-GLAS-METAL")
+                elif "papir" in raw_type:
+                    fraktion = "Pap- og Papiraffald"
+                    icon = ICON_MAP.get("PAP-PAPIR")
+
+                if cells[3].get_text() is not None and cells[3].get_text() != "":
+                    current_pickup = datetime.strptime(cells[3].get_text(), "%d-%m-%Y").date()
+                    next_pickup = datetime.strptime(cells[4].get_text(), "%d-%m-%Y").date()
+
+                    if current_pickup == datetime.now().date():
```
Comment (5ila5) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/renodjurs_dk.py:68]: This if is not necessary, it's fine to return past collections

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/renodjurs_dk.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+                    fraktion = "Rest- og Madaffald"
+                    icon = ICON_MAP.get("REST-MAD")
+                elif "plast" in raw_type and "glas" in raw_type:
+                    fraktion = "Plast-, Glas- og Metalaffald"
+                    icon = ICON_MAP.get("PLAST-GLAS-METAL")
+                elif "papir" in raw_type:
+                    fraktion = "Pap- og Papiraffald"
+                    icon = ICON_MAP.get("PAP-PAPIR")
+
+                if cells[3].get_text() is not None and cells[3].get_text() != "":
```
Comment (5ila5) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/renodjurs_dk.py:64]: Same as above: `if cells[3].get_text() is None or cells[3].get_text() == "": continue`

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/renodjurs_dk.py (hunk 2 above)
Comment (MTrab) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/renodjurs_dk.py:68]: It only shows current and next - there's no collections of any kind, so this is needed

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/renodjurs_dk.py (hunk 1 above)
Comment (MTrab) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/renodjurs_dk.py:47]: Guess you mean `<=` and not `>=`

**Comment by 5ila5** [comment]
Thanks for your contribution.

Did some reformatting (based on .pre-commit-config.yaml)
and run the update_docu_links.py script

**Pull request merged**